x11-libs/kimageannotator/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>dflogeras2@gmail.com</email>
+		<name>David Flogeras</name>
+	</maintainer>
+	<longdescription lang="en">Tool for annotating images.
+	</longdescription>
```

Comment:
Same as `DESCRIPTION`, so does not make sense to duplicate here. Also flagged by CI.